x-pack/legacy/plugins/apm/public/components/app/ServiceDetails/ServiceIntegrations/index.tsx

```diff
@@ -148,7 +148,9 @@ export class ServiceIntegrations extends React.Component<Props, State> {
                 panels={[
                   {
                     id: 0,
-                    items: this.getPanelItems(license.features.ml?.is_available)
+                    items: this.getPanelItems(
+                      license.getFeature('ml').isAvailable
```

Comment:
Will `getFeature` always return an object regardless if the plugin is disabled?

Reply:
Yes. `getFeature` always returns a `LicenseFeature`: https://github.com/elastic/kibana/blob/88ae8d07c8ad68352db952e229b7b3b73bfb9ae3/x-pack/plugins/licensing/common/types.ts#L30-L33

Reply:
We're using `ILicense | undefined` now so a `?` has been added to `license`, but the above still holds.